.github/workflows/node.js.yml

```diff
@@ -10,16 +10,49 @@ on:
         branches: [master]
 
 jobs:
+    clone_vector_skin:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Clone vector skin repository"
+              run: |
+                  git clone "https://gerrit.wikimedia.org/r/mediawiki/skins/Vector" skins/Vector
+
+    mediawiki_docker:
+        env:
+            MW_DOCKER_PORT: 8080
+            MW_SCRIPT_PATH: /w
+            MW_SERVER: http://localhost:8080
+            MEDIAWIKI_USER: Admin
+            MEDIAWIKI_PASSWORD: dockerpass
+            XDEBUG_CONFIG: ""
+        runs-on: ubuntu-latest
+        needs: clone_vector_skin
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Setup mediawiki-docker"
+              run: |
+                  docker-compose up -d
+                  docker ps -a
+                  docker-compose exec -T mediawiki composer update
+                  docker-compose exec -T mediawiki /bin/bash /docker/install.sh
+            - name: "Add vector skin config to LocalSettings.php file"
+              run: |
+                  echo "wfLoadSkin( 'Vector' );" >> LocalSettings.php
```

Comment:
Try adding a `sudo` here and see if it works

Reply:
Sure. That works. Thanks